content/docs/sidebar.json

```diff
@@ -111,6 +111,21 @@
             "slug": "docker",
             "label": "Docker",
             "source": "docker.md"
+          },
+          {
+            "slug": "requirements",
+            "label": "Requirements",
+            "source": "requirements.md"
+          },
+          {
+            "slug": "venv",
+            "label": "Virtual Environments",
```

Comment:
I think this should not be plural - i.e. `Environment`. Doesn't affect `Requirements` though. If you're going to apply to the other pages in the #188, would be great. E.g. `Python packages` > `Python package`.

Reply:
done, I will look at #188 separately, because lots of changes to be done in this PR itself